packages/cloudflare/test/sdk.test.ts

```diff
@@ -0,0 +1,13 @@
+import { describe, expect, test, vi } from 'vitest';
+
+import * as SentryCore from '@sentry/core';
+import { init } from '../src/sdk';
+
+describe('init', () => {
```

Comment:
m: could we add a test for init returning a client?

Reply:
Changed with 87b4fa1b8959cf69a8f48419ea6147772666edb7